lib/src/test/java/growthbook/sdk/java/GBFeaturesRepositoryTest.java

```diff
@@ -159,14 +234,14 @@ private static OkHttpClient mockHttpClient(final String serializedBody) throws I
         Call remoteCall = mock(Call.class);
 
         Response response = new Response.Builder()
-            .request(new Request.Builder().url("http://url.com").build())
-            .protocol(Protocol.HTTP_1_1)
-            .code(200).message("").body(
-                ResponseBody.create(
-                    serializedBody,
-                    MediaType.parse("application/json")
-                ))
-            .build();
+                .request(new Request.Builder().url("http://url.com").build())
+                .protocol(Protocol.HTTP_1_1)
+                .code(200).message("").body(
+                        ResponseBody.create(
+                                serializedBody,
+                                MediaType.parse("application/json")
+                        ))
+                .build();
```

Comment:
Can you clean up these unnecessary changes?

Reply:
sorry it's because of my auto style code